### PR TITLE
Return in viewState.load() if iModel associated with viewState is a blankConnection

### PR DESCRIPTION
**Pull request by nick4598**
The addition of the RPC operation "hydrateViewState" causes load() to now fail when the iModel associated with the viewState is a blankConnection. This PR returns if the iModel is a blank connection to avoid this problem. 

Before the RPC operation hydrateViewState, load() likely failed gracefully in the case of a BlankConnection.

### CODE REVIEW

**Review by pmconne — APPROVED** [review]

**Comment by pmconne** [comment]
@Mergifyio backport release/3.2.x

File: core/frontend/src/ViewState.ts [hunk 1]
```diff
@@ -350,6 +350,11 @@ export abstract class ViewState extends ElementState {
...
    */
   public async load(): Promise<void> {
+    // If the iModel associated with the viewState is a blankConnection,
+    // then no data can be retrieved from the backend.
+    if (this.iModel.isBlank)
+      return;
+
     const hydrateRequest: HydrateViewStateRequestProps = {};
     this.preload(hydrateRequest);
     const promises = [
```
Comment (MarcBedard8) [inline core/frontend/src/ViewState.ts:360]: With this fix, displayStyle.load will never be called. Is it what is expected? I mean, where is the saved view state will actually be restored for blank imodel connection?

Comment (pmconne) [reply core/frontend/src/ViewState.ts:360]: DisplayStyle.load, like ViewState.load, can't do anything useful in the absence of a backend. Recalling a saved view restores not only the view itself but also its display style, model/category selectors, etc. ViewState.load is invoked *after* this state has been restored.

Comment (pmconne) [reply core/frontend/src/ViewState.ts:360]: It looks like @MarcBedard8 deleted his comment.

**Comment by pmconne** [comment]
Force-merging because unrelated appui test hang on Windows.

**Pull request merged**